pysigma/signatures.py

```diff
@@ -136,12 +137,25 @@ def normalize_field_map(field: Dict[str, Any]) -> DetectionMap:
         if value is None:
             out[key] = [None], modifiers
         elif isinstance(value, list):
-            out[key] = [
-                apply_modifiers(str(_v), modifiers) if _v is not None else None
-                for _v in value
-            ], modifiers
+            if key not in out:
```

Comment:
Could we check for the key overlap a few lines up? Do the checks one after the other rather than nested ifs that add more code?